src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -87,6 +87,17 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ åpenBehand
     const visSubmitKnapp =
         !vurderErLesevisning() && åpenBehandling?.status === BehandlingStatus.UTREDES;
 
+    const [visTrekkILøpendeUtbetaling, settVisTrekkILøpendeUtbetaling] = React.useState(false);
+    const [erUlagretNyTrekkILøpendeUtbetaling, settErUlagretNyTrekkILøpendeUtbetaling] =
+        React.useState(false);
+
+    React.useEffect(() => {
+        settVisTrekkILøpendeUtbetaling(
+            !!åpenBehandling.trekkILøpendeUtbetaling &&
+                !!toggles[ToggleNavn.trekkILøpendeUtbetaling]
```

Comment:
Usikker på om vi trenger !! før toggle

Reply:
dette er for å caste til boolean fra noe som kan være undefined, tror jeg

Reply:
Jepp, men tok en titt på det nå og ser ikke ut som toggles[ToggleNavn.trekkILøpendeUtbetaling] kan være undefined, den blir boolean når jeg trekker den ut hvertfall (og intellij klager ikke når jeg fjerner !!)